src/app.js

```diff
@@ -1,10 +1,6 @@
-function app() {
-  const invite = 'Hello from app.js';
-  console.log(invite);
-}
-export default app
+import createBtn from './btn';
 
-import createBtn from "./btn";
-
-const btn = createBtn
-btn()
\ No newline at end of file
+const root = document.querySelector('#app');
+const btn = createBtn('play the game');
+root.appendChild(btn);
+btn();
```

Comment:
remove this `btn()` call - you store the return value of `createBtn()`call which is not callable (it's not a function).

Reply:
left over from a previous attempt sory ;D

Reply:
Why have you deleted the `app` function declaration and it's export?